### DISCUSSION TITLE
Xcode 16 now supports some EditorConfig properties

### DISCUSSION THREAD

**Discussion by mokagio** [editorconfig/editorconfig#517] [General]
Apple's Xcode 16 IDE shipped support for a handful of properties.

As per [the release notes](https://developer.apple.com/documentation/xcode-release-notes/xcode-16-release-notes), the properties are:

- `insert_final_newline`
- `trim_trailing_whitespace`
- `indent_style`
- `indent_size`
- `tab_width`
- `end_of_line`
- `max_line_length`

Thanks!

See also https://x.com/mokagio/status/1839809925570265100 and https://gist.github.com/mokagio/bd400f790bcb06a79fdf2a837dbce0ce

**Comment by xuhdev** [comment]
Thanks for notifying us! This is being tracked at https://github.com/editorconfig/editorconfig.github.com/issues/185.
